client/test/components/notebook/convert.test.ts

```diff
@@ -0,0 +1,250 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
```

Comment:
The location of this file seems off. If my test lives at `client/test/components/notebook/convert.test.ts`, my expectation would be at `client/src/components/notebook/convert.ts`

Reply:
done in 5f3a455e8798a4a89997b40e894f660e63b00135